# When to start the camera placement trajectory
# (There is no header here because the frame is specified per-camera-placement.)
time stamp

# The vector of camera placements
rviz_view_controllers/CameraPlacement[] placements

# The interaction style that should be activiated when movement is done.
uint8 mouse_interaction_mode
uint8 FPS   = 0 # Acts like the FPS controller
uint8 ORBIT = 1 # Acts like the Orbit controller



